docs/docs/app/datasources.mdx

```diff
@@ -294,3 +294,24 @@ We require 4 things to query the Universal Analytics API:
 4.  Custom Dimension Delimiter (defaults to `:`)
 
 When tracking experiment views, the custom dimension value must be formatted as `{experiment-key}{delimiter}{variation-index}`. For example: `my-test:0` for the control and `my-test:1` for the 1st variation.
+
+### Schema Browser
```

Comment:
I think this should be an H2 instead of H3. Right now it's under connection settings, which doesn't seem to fit